### ISSUE TITLE
Problem after start test miner.

### ISSUE THREAD

**Issue by aleksfox** [nginnever/zogminer#14]
[ 1084.587016] VM fault (0x01, vmid 3) at page 47326788, read from 'TC0' (0x54433000) (8)
[ 1084.587016] amdgpu 0000:07:00.0: GPU fault detected: 147 0x05708801
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_ADDR 0x00E0983C
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_STATUS 0x06088002
[ 1084.587016] VM fault (0x02, vmid 3) at page 14719036, read from 'TC6' (0x54433600) (136)
[ 1084.587016] amdgpu 0000:07:00.0: GPU fault detected: 147 0x0d78c401
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_ADDR 0x03B9780A
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_STATUS 0x060C4001
[ 1084.587016] VM fault (0x01, vmid 3) at page 62486538, read from 'TC3' (0x54433300) (196)
[ 1084.587016] amdgpu 0000:07:00.0: GPU fault detected: 147 0x0c88c801
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_ADDR 0x016D8DF6
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_STATUS 0x06044001
[ 1084.587016] VM fault (0x01, vmid 3) at page 23956982, read from 'TC5' (0x54433500) (68)
[ 1084.587016] amdgpu 0000:07:00.0: GPU fault detected: 147 0x09e08801
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_ADDR 0x006F7D1B
[ 1084.587016] amdgpu 0000:07:00.0: VM_CONTEXT1_PROTECTION_FAULT_STATUS 0x060C4002
etc. etc...

**Comment by omaralvarez** [comment]
I will look into this whenever I can. I may need further info about how the error happened.

**Comment by aleksfox** [comment]
What kind of information do you need?
I am a novice user in Ubuntu, but I believe that I can provide all the information you ask
Ubuntu 16.04.1 LTS
2xSapphire RX470 8Gb
AMDGPU-Pro Driver Version 16.30 for Ubuntu 16.04
sudo apt-get update\upgrade after install
upgrade kernel to stable: 4.8.4

and i try to do this (because it had problems with the operation of the system):
http://askubuntu.com/questions/761706/ubuntu-15-10-and-16-04-keep-freezing-randomly
https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1511019/comments/8

**Comment by aleksfox** [comment]
From z.cash forum:
The reason why the RX 470/480 crashes is because you all have updated the bios with the 1500 Strap and while that works for ETH it might not work for ZCASH. So flash back firmware to stock.

Can you confirm it?

**Comment by omaralvarez** [comment]
Yep, overclocking can mess you up. What works for ether can crash your rigs with these miners. Totally different algo, I will be testing on reference fury's today or tomorrow.

**Comment by d3strange** [comment]
I reflashed the stock ROM on 4GB rx470 - same issue.  I don't believe it is related to overclocking.

**Comment by aleksfox** [comment]
Try to reflash to stock rom... i have no freeze right now, but after miner running about 5-7 minute a have this: "terminate called after throwing an instance of 'boost::exception_detail::clone_impl<boost::exception_detail::error_info_injector<boost::system::system_error> >'
  what():  wait: Interrupted system call
Aborted (core dumped)
"

**Comment by omaralvarez** [comment]
Stratum support is not yet complete. And bear in mind the miner is not yet finished. We will look into this when we can.

**Comment by aleksfox** [comment]
OK. For now, i'm just update the message as detecting problems to make it easier to understand then.

**Comment by omaralvarez** [comment]
Thanks for the testing!
